src/App.tsx

```diff
@@ -1,24 +1,51 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+import { Clock } from './components/Clock/Clock';
+import { Button } from './components/Button';
+
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    isClockVisible: true,
+    clockName: 1,
+  };
+
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  };
+
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  };
+
+  randomName = () => {
```

Comment:
что randomName
get? set? change?

функция должна начинать свое название на что сделать
установитьРандомноеИмя
получитьРандомноеИмя
изменитьРандомноеИмя

```suggestion
  setRandomName = () => {
```